src/components/Keyboard.js

```diff
@@ -16,7 +48,7 @@ export default function Keyboard({ addLetters, deleteLetters }) {
                 <button className="keyboard-letter" onClick={() => addLetters('I')}>I</button>
```

Comment:
There's a lot of repeat code here. That's what React is for!

```tsx
const KeyboardButton = ({ letter, setter }) => 
<button className="keyboard-letter" onClick={() => setter(letter)}>{letter}</button>
```

```tsx
<KeyboardButton letter={A} setter={addLetters} />
<KeyboardButton letter={B} setter={addLetters} />
<KeyboardButton letter={C} setter={addLetters} />
<KeyboardButton letter={D} setter={addLetters} />
```

Hold up... still seems like we're repeating things...

```tsx
const alphabet = ["A", "B", "C", "D", ...];

...

return alphabet.map(letter => 
   <KeyboardButton key={letter} letter={letter} setter={addLetters} />
)
```